plugins/woocommerce/tests/e2e-pw/tests/merchant/settings-woo-com.spec.js

```diff
@@ -3,7 +3,13 @@ const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
 
 test.describe(
 	'WooCommerce woo.com Settings',
-	{ tag: [ '@services', '@skip-on-default-pressable' ] },
+	{
+		tag: [
+			'@services',
+			'@skip-on-default-pressable',
+			'@skip-on-default-wpcom',
```

Comment:
Getting error 404. Will address it in a separate PR.

Reply:
For the WPCOM env, it is not testable because the Analytics there is turned on by default and can't be turned OFF for the test case inside (no switch in WooCommerce UI) - so we should skip it. But it should work for Pressable.